src/workerd/api/actor-state.c++

```diff
@@ -122,7 +122,8 @@ ActorObserver& currentActorMetrics() {
 
 jsg::JsRef<jsg::JsValue> listResultsToMap(
     jsg::Lock& js, ActorCacheOps::GetResultList value, bool completelyCached) {
-  return js.withinHandleScope([&] {
+  return js
+      .withinHandleScope([&] {
```

Comment:
This change seem unnecessary

Reply:
This was the `just f` formatter for some odd reason. Really not sure why it did this.

Reply:
Yeah, just confirmed again, `just f` is doing this and complains about it if I change it back.